src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/AppProvisioningTool.cs

```diff
@@ -186,6 +203,32 @@ public AppProvisioningTool(string commandName, ProvisioningToolOptions provision
             return effectiveApplicationParameters;
         }
 
+        private async Task<ApplicationParameters?> CreateAppRegistration(TokenCredential tokenCredential, ApplicationParameters? applicationParameters)
```

Comment:
Is this the same as the type returned by 'list applications for tenant' (--list-aad-apps)?

Reply:
It is not and that was a big question that I wanted to address. It's a custom object that we have been using. It has all the important info but I imagine for parsing, using Microsoft.Graph.Application might be the way to go. I'll have a quick chat with Zach but it should align with --list-aad-apps

Reply:
changed it to Microsoft.Graph.Application. consisten with --list-aad-apps